guess_number/guess_word.py

```diff
@@ -0,0 +1,25 @@
+"""Pick a random word and have the user try to guess it. Provide feedback on the guessed word"""
+import random
+
+words = ["a", "be", "cat", "dog", "elf"]
```

Comment:
This appears to be a constant so I'd make it uppercase (and choose a more descriptive name).